Assets/MRTK/SDK/Features/UX/Scripts/BoundsControl/BoundsControl.cs

```diff
@@ -1235,16 +1250,15 @@ private void TransformTarget(HandleType transformType)
                     // If non-uniform scaling or uniform scaling only on the non-flattened axes
                     if (ScaleHandlesConfig.ScaleBehavior != HandleScaleMode.Uniform || !UniformScaleOnFlattenedAxis)
                     {
-                        FlattenModeType determinedType = FlattenAxis == FlattenModeType.FlattenAuto ? VisualUtils.DetermineAxisToFlatten(TargetBounds.bounds.extents) : FlattenAxis;
-                        if (determinedType == FlattenModeType.FlattenX)
+                        if (actualFlattenAxis == FlattenModeType.FlattenX)
```

Comment:
Could `actualFlattenAxis` be cached here, similar to how it was before? That way we're not calling into the property (and thus potentially calling into `VisualUtils.DetermineAxisToFlatten`) every time? Or do we expect that each call here might be slightly different (and/or the call is negligible perf-wise)?

Reply:
Ah, good catch. The perf is negligible but every little bit matters! Fixed, it's now cached for each entry in those if-statements.